api/osrd_infra/models/rolling_stock.py

```diff
@@ -62,6 +62,14 @@ class RollingStock(models.Model):
     )
     loading_gauge = models.CharField(max_length=16, choices=[(x.value, x.name) for x in LoadingGaugeType])
     image = models.ImageField(null=True, blank=True)
+    meta_data = models.JSONField(
```

Comment:
meta_data is usually written `metadata`, as [a single word](https://en.wikipedia.org/wiki/Metadata)